cypress/component/rich-text/document-mocks/normalizationWithoutValueChange.js

```diff
@@ -102,6 +102,12 @@ export default {
             },
           ],
         },
+      ],
+    },
+    {
+      nodeType: 'unordered-list',
+      data: {},
+      content: [
```

Comment:
Previously an empty list was being dropped instead of being fixed (by adding an empty list item) like the other cases in the same test 🤷🏻